README.md

```diff
@@ -27,20 +27,24 @@ This project is the frontend of Gordon 360 in React. [The retired frontend](http
 
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app). Read the user guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
 
-Make sure Node.js is set up on your machine. The following procedures will install nvm, a node.js version manager onto your machine. An npm installation may also work, but nvm will allow easier installation and use of this particular version.
+Make sure Node.js is set up on your machine. If you are on the CS-RDSH-02 virtual machine, it should be setup already. To check if it is on a machine, open a terminal and run the command `nvm`. If the output says `Running version...`, then Node.js is setup. If not, the following procedures will install nvm, a node.js version manager onto your machine. An npm installation may also work, but nvm will allow easier installation and use of this particular version.
```

Comment:
> If the output says `Running version...`, then Node.js is setup.
This is only true for the `nvm-windows` package. On Linux and MacOS (or any other shell terminal), running nvm will give you a very  verbose description of the nvm CLI and how to use it. I think it might be clearer to say that if the terminal returns ` 'nvm' is not recognized...`, then it needs to be installed.